frontend/src/shared/components/baseCard/baseCard.css

```diff
@@ -48,20 +46,16 @@
 
 .card-content {
     --card-text-width: 420px;
-
     display: block;
     width: 100%;
-    min-height: 1px;  /*DO NOT REMOVE THIS LINE. OTHERWISE YOU WILL EXPERIENCE PAIN IN TRYING TO FIX COLUMN STACKING ON CHROME/IOS*/
+    min-height: 1px;
+    /*DO NOT REMOVE THIS LINE. OTHERWISE YOU WILL EXPERIENCE PAIN IN TRYING TO FIX COLUMN STACKING ON CHROME/IOS*/
```

Comment:
This is now ambiguous what line not to remove

Reply:
somehow the code formatter decide to change this one. I will revert it

Reply:
I should make that comment more descriptive sometime :P